### PR TITLE
GP2-1179 add category support to dropdown

### PR DESCRIPTION
**Pull request by dahfool**
Add category support to dropdown

_Tick or delete as appropriate:_

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse//GP2-1179
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

**Comment by dahfool** [comment]
image

File: react-components/src/components/Form/Select/index.jsx [hunk 1]
```diff
@@ -96,7 +116,6 @@ export const Select = ({
           isOpen ? 'select__button--close' : ''
         }`}
         type="button"
-        role="button"
```
Comment (bobmeredith) [inline react-components/src/components/Form/Select/index.jsx:99]: You're removing this role as it's already a button so the role is redundant?

**Review by bobmeredith — APPROVED** [review]
LGTM - just one tiny question

File: react-components/src/components/Form/Select/index.jsx (hunk 1 above)
Comment (dahfool) [reply react-components/src/components/Form/Select/index.jsx:99]: > You're removing this role as it's already a button so the role is redundant?

Yes redundant, i was gettingt this warning from eslint - The element button has an implicit role of button. Defining this explicitly is redundant and should be avoided.(jsx-a11y/no-redundant-roles)

**Pull request merged**